src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,38 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Enkäten på svenska', en: 'Questionnaire in English'};
+    const languageOptionsArray = config.languages.map((lang) => {
+      if (getAttr(hearing.title, lang, {exact: true}) && lang === activeLanguage) {
```

Comment:
In the same hearing with two languages, not all the UI elements are rendered in English even if English is selected. "Mielipiteet" part is still in Finnish.
![image](https://cloud.githubusercontent.com/assets/9367712/21892005/0255027a-d8de-11e6-9ecf-8b7e9b8e5b43.png)